utils/rpms/daos.spec

```diff
@@ -283,6 +283,7 @@ getent passwd daos_agent >/dev/null || useradd -s /sbin/nologin -r -g daos_agent
 %dir %{_sysconfdir}/bash_completion.d
 %{_sysconfdir}/bash_completion.d/daos.bash
 %{_libdir}/libdaos_common.so
+%{_libdir}/libdaos_common_pmem.so
```

Comment:
should we have this in server and the other library in client rather than in common list ?

Also, need to address the question about debian

Reply:
Since its just pmem and non-pmem variant I don't think this is specific to server. Thats why I wanted to keep both in the same place.